src/data/text/item_descriptions.h

```diff
@@ -2750,9 +2780,9 @@ static const u8 sAdrenalineOrbDesc[] = _(
     "but only one time.");
 
 static const u8 sHoneyDesc[] = _(
-    "A sweet honey that\n"
-    "attracts wild Pokémon\n"
-    "in grass or on trees.");
+    "Sweet honey. It can\n"
+    "be sold to shops at\n"
+    "a low price.");
```

Comment:
i think this should be behind item config (since its effect was changed from sweet scent to sellable item in gen 8)

Reply:
I was sticking to SwSh's because I couldn't get a good enough shortening of the Honey's old effect, and well, because the old effect is still missing from the item_expansion, meaning if someone already had to go out of their way to implement it, why not have them fix the item's description as they saw fit too? :eyes:

Reply:
I think the original effect of Honey is in scope of item expansion (it'd just use sweet scent anyways so shouldn't be that bad) and should be implemented

Reply:
Huh. Alright then, I'm opened to suggestions for its old description.

Also, maybe opening a PR as a reminder would be a good idea.